data/acceptable_tekton_bundles.yml

```diff
@@ -265,21 +265,21 @@ pipeline-bundles:
       tag: d2e1cbe0cab869ade97d9b6c7235a23797e9b119
 pipeline-required-tasks:
   nodejs:
```

Comment:
How come nodejs is the only one here?

Reply:
Good question. I'm wondering if that's the only pipeline that's been updated since the track bundle changes went in?

Reply:
I doubt it... Usually all the pipelines are always updated. You can confirm this by looking at the recent PRs from build-definitions coming to this repo.

Reply:
something may be wrong then. I'll check it out. thanks.